syntax-highlighting-code-block.php

```diff
@@ -222,3 +233,71 @@ function render_block( $attributes, $content ) {
 		);
 	}
 }
+
+/**
+ * Initialize admin settings.
+ */
+function admin_init() {
+	register_setting( 'syntax_highlighting', 'syntax_highlighting_options' );
+}
+add_action( 'admin_init', __NAMESPACE__ . '\admin_init' );
+
+/**
+ * Sanitize the given theme name to make sure it's a valid highlight.js theme.
+ *
+ * @param string $input Value of the dropdown.
+ *
+ * @return string The sanitized theme name
+ */
+function sanitize_theme_name( $input ) {
+	require_once __DIR__ . '/vendor/scrivo/highlight.php/HighlightUtilities/functions.php';
+
+	$themes = \HighlightUtilities\getAvailableStyleSheets();
+
+	return in_array( $input, $themes, true ) ? sanitize_text_field( $input ) : 'default';
+}
+
+/**
+ * Add plugin settings to Customizer.
+ *
+ * @param \WP_Customize_Manager $wp_customize The Customizer object.
+ */
+function customize_register( $wp_customize ) {
+	require_once __DIR__ . '/vendor/scrivo/highlight.php/HighlightUtilities/functions.php';
+
+	$themes  = \HighlightUtilities\getAvailableStyleSheets();
+	$choices = array_combine( $themes, $themes );
+
+	$style = get_option(
+		'syntax_highlighting_options',
+		[
+			'theme_name' => 'default',
+		]
+	)['theme_name'];
+
+	$wp_customize->add_section(
```

Comment:
As opposed to creating a new section, what about reusing `colors` if it exists?

Reply:
Is there a way to check if the `colors` section exists? I tried looking at `$wp_customize->sections()`, however, that array isn't populated with `colors` at the time the plugin's customizer registration occurs.